examples/sampleProp.conf

```diff
@@ -19,4 +19,21 @@ IotCCPassword = "Password"
 
 GraphiteIP = "Graphite-IP"
 GraphitePort = None
- 
+
+#### [MQTT] ####
+MqttChannel = "mqtt-channel"
+MqttSubChannel1 = "mqtt-subchannel1"
+BrokerIP = "127.0.0.1"
```

Comment:
Keep it general 
BrokerIP = mqtt-broker
BrokerPort = mqtt-broker-port

I'll suggest to make it a different config file as samplePropMqtt.conf rather then keeping all this parameters in the main sampleProp.conf file.